### DISCUSSION TITLE
Dispatch public roadmap

### DISCUSSION THREAD

**Discussion by Pryz** [dispatchrun/dispatch#1] [Feature Requests]
tHi there!

We have created a public roadmap for you to see what's coming for Dispatch. Please take a look and feel free to add any new feature requests!

Learn more about our product roadmap at https://docs.dispatch.run/company/roadmap/

**Comment by Tyl13** [comment]
It looks like the roadmap is now at this web page:
https://docs.dispatch.run/company/roadmap/

**Reply by Pryz** [reply]
Fixed :) thanks!
